mkosi

```diff
@@ -82,16 +86,19 @@ def print_error(text: str) -> None:
 def run(cmdline: List[str], execvp: bool = False, **kwargs: Any) -> subprocess.CompletedProcess:
     if 'run' in arg_debug:
         sys.stderr.write('+ ' + ' '.join(shlex.quote(x) for x in cmdline) + '\n')
-    if execvp:
-        assert not kwargs
-        os.execvp(cmdline[0], cmdline)
-    else:
-        return subprocess.run(cmdline, **kwargs)
+    try:
+        if execvp:
+            assert not kwargs
+            os.execvp(cmdline[0], cmdline)
+        else:
+            return subprocess.run(cmdline, **kwargs)
+    except FileNotFoundError:
+        raise MkosiCommandException(f"{cmdline[0]} not found in PATH.")
```

Comment:
Unfortunately I cannot seem to make a multiline suggestion, but a nicer wrapping would be
```python
except FileNotFoundError as e:
    raise MkosiCommandException(f"{cmdline[0]} not found in PATH.") from e
```
This does not immediately affect anything, because the exception is caught in main, but I've always thought about reusing mkosi as a library and there this would lead to a more informative stack trace.